analyticsdata/test/analyticsDataTest.php

```diff
@@ -51,6 +51,22 @@ public function testClientFromJsonCredentials()
         }
     }
 
+    public function testGetCommonMetadata()
+    {
+        $propertyId = self::requireEnv('GA_TEST_PROPERTY_ID');
+        $output = $this->runFunctionSnippet('get_common_metadata', [$propertyId]);
```

Comment:
The tests are failing because `get_common_metadata()` doesn't take an argument
```suggestion
        $output = $this->runFunctionSnippet('get_common_metadata');
```